tools/pkg/engine_build_configs/lib/src/build_config_runner.dart

```diff
@@ -625,10 +633,15 @@ final class BuildRunner extends Runner {
       if (dryRun) {
         processResult = _dryRunResult;
       } else {
+        final bool shouldEmitAnsi =
+            io.stdout.supportsAnsiEscapes || io.Platform.environment['CLICOLOR_FORCE'] == '1';
```

Comment:
There should be a way to disable color even when attached to an ansi terminal. It doesn't have to be plumbed all the way to the `et` command line in this PR, but it should at least be exposed, for example as an argument to the constructor of the containing class.

Reply:
Done, I made the negation turn off colors too despite them being supported.  I'm not sure of the use-case where they are supported but someone wouldn't' want them.  Usually they are disabled to support parsing.